NuGet.Config

```diff
@@ -5,7 +5,7 @@
     <!-- Do not add any additional feeds if new packages are needed they need to come from nuget.org or our azure-sdk-for-net DevOps feed -->
     <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
     <add key="azure-sdk-for-net" value="https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-net/nuget/v3/index.json" />
-
+    <add key="azure_app_service" value="https://www.myget.org/F/azure-appservice/api/v2" />
```

Comment:
This PR is in draft mode because the required version of Microsoft.Azure.WebJobs hasn't been published to nuget yet. The config change here is temporary to get CI passing. The rest of this PR can be reviewed. Before signoff/checkin, I'll update this PR to reference the official packages once they're published.